smsclient_ovh/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+===============
+SMS client OVH
+===============
+
+This module provide OVH fields to create OVH gateway.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to settings > technical > telephony >gateway configuration and select OVH http in method
+* go to settings > technical > keychain and set your credentials
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
```

Comment:
use short description of bug tracker (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)